ahcore/utils/callbacks.py

```diff
@@ -27,6 +29,41 @@
 logging.getLogger("pyvips").setLevel(logging.ERROR)
 
 
+class NormalizationType(str, Enum):
+    SIGMOID = "sigmoid"
+    SOFTMAX = "softmax"
+    LOGITS = "logits"
+
+    def normalize(self):
+        if self == NormalizationType.SIGMOID:
+            return torch.sigmoid
+        elif self == NormalizationType.SOFTMAX:
+            return torch.softmax
+        elif self == NormalizationType.LOGITS:
+            return lambda x: x
+        else:
+            raise ValueError("Function not supported")
+
+
+class InferencePrecision(str, Enum):
+    FP16 = "float16"
+    FP32 = "float32"
+    UINT8 = "uint8"
+
+    def __str__(self) -> str:
```

Comment:
Is this necessary? When subclassing both to str and Enum, doesn't `str(InferencePrecision.FP16)` already return the string value?